frontend/src/app/modules/boards/board/board-partitioned-page/board-list-container.component.ts

```diff
@@ -118,6 +122,33 @@ export class BoardListContainerComponent extends UntilDestroyedMixin implements
     });
   }
 
+  checkForListErrors(board:Board) {
+    const newColumnsQueryProps:any = {
+      'columns[]': ['id', 'subject'],
+      'showHierarchies': false,
+      'pageSize': 500,
+      'filters': {}
+    };
+    board.queries.forEach((listQuery) => {
+     newColumnsQueryProps.filters = JSON.stringify(listQuery.options.filters);
+
+      let observable = this
+      .apiv3Service
+      .queries
+      .find(newColumnsQueryProps , listQuery.options.queryId as string);
+
+      observable
+      .subscribe(
+        query => {this.wpStatesInitialization.updateQuerySpace(query, query.results); },
+        (error) => {
```

Comment:
I think that here we should handle only errors due to the lack of permissions of the user. It seems to me that other types of errors should be communicated to the user.